index.html

```diff
@@ -41,30 +41,30 @@ <h2>Why you should use it</h2>
 		</div>
 		<div class="row">
 			<div class="col-sm-4 feature">
-				<h3>Always Connected</h3>
-				<p>The Lounge never sleeps. Just sign in and pick up where you left off.</p>
+				<h3>Modern features</h3>
+				<p>Push notifications, link previews, file uploads, and more bring IRC to the 21st century.</p>
 			</div>
 			<div class="col-sm-4 feature">
-				<h3>Responsive Interface</h3>
-				<p>The client works smoothly on every desktop, smartphone and tablet.</p>
+				<h3>Always connected</h3>
+				<p>Remains connected to IRC servers while you are offline. Forget about bouncers. Always resume where you left off no matter what device.</p>
```

Comment:
```suggestion
				<p>Remain connected to IRC servers while you are offline. Forget about bouncers. Resume where you left off on any device.</p>
```

("no matter what device" requires a verb, like "no matter what device you use", etc.)